### DISCUSSION TITLE
How to make multiple characters sing

### DISCUSSION THREAD

**Discussion by DuskDN75** [Leather128/LeathersFunkinEngine#195] [Q&A]
Hi, how do you make multiple characters sing in a song?

**Comment by ghost** [comment]
1. go to chart menu and the go to chart
2. go find Character Group
3. then you need to put a number to the group 
### like example:

- 0 = first character
- 1 = second character

and if you need a character to sing together
this is to sing together : 

- 0,1

and done.

**Reply by DuskDN75** [reply]
I dont quite understand, also I'm trying to make the opponent and gf sing.

**Reply by ghost** [reply]
oh well i don't know how to do that opponent and gf sing together
sorry

**Reply by DuskDN75** [reply]
Oh, alr

**Reply by DuskDN75** [reply]
Well, do you know how to add in 2 characters?

**Reply by ghost** [reply]
Yes, it is simple is go like this:
![Untitled](https://user-images.githubusercontent.com/87716423/156480409-8aea0177-4170-4393-948a-c38ede5ae7d0.png)

**Reply by DuskDN75** [reply]
ohh k, in the song data file?

**Reply by DuskDN75** [reply]
Also is there a way to change the size of the strum? Its too small for me

**Reply by cudadotsln** [reply]
The file location isn't in the song data file, its in the characters folder.

**Comment by ghost** [comment]
I just started looking here lol
If you still don't understand, in Psych Engine there is a note option for gf to sing in it.
If you know how to like, code that in or smth, that can work.
Another thing is, you can make a character group with your opponent and gf, just with gf having an offset to her right position.
